Maths/fibonacciSeries.py

```diff
@@ -1,16 +1,18 @@
 # Fibonacci Sequence Using Recursion
 
 def recur_fibo(n):
-   if n <= 1:
-       return n
-   else:
-       return(recur_fibo(n-1) + recur_fibo(n-2))
+    return n if n <= 1 else (recur_fibo(n-1) + recur_fibo(n-2))
     
-limit = int(input("How many terms to include in fibonacci series: "))
+def isPositiveInteger(limit):
+    return limit >= 0
 
-if limit <= 0:
-   print("Please enter a positive integer: ")
-else:
-   print(f"The first {limit} terms of the fibonacci series are as follows")
-   for i in range(limit):
-       print(recur_fibo(i))
+def main():
+    limit = int(input("How many terms to include in fibonacci series: "))
+    if isPositiveInteger:
```

Comment:
I think you should write:
```python
if isPositiveInteger(limit):
```
Otherwise, It will not show the error message when provided a negative number.
Following is the output of your current script when you provide a negative number:
```python
How many terms to include in fibonacci series:  -7
The first -7 terms of the fibonacci series are as follows:
[]
```
Instead it should display something like:
```python
How many terms to include in fibonacci series:  -7
Please enter a positive integer.
```